std/getopt.d

```diff
@@ -1632,6 +1635,13 @@ void defaultGetoptPrinter(string text, Option[] opt)
 in the manner described in the documentation of function
 `defaultGetoptPrinter`.
 
+The following prints `text` and `Option`s `opt` on `stderr`:
+
+------------
+import std.stdio : stderr;
+defaultGetoptFormatter(stderr.lockingTextWriter, text, opt);
```

Comment:
Can you move this to documented unittest?

Reply:
You mean the existing one on row 445 or a new one?

I was actually about to close this PR as I reverted from doing this in my code. Now I think it is better to print an error on `stderr` referencing "-h", and continue printing the help on `stderr`.

Do you agree on closing, or do you find the PR helpful?